src/components/Dashboard/Welcome/Welcome.tsx

```diff
@@ -2,7 +2,6 @@ import React, { ReactElement } from 'react';
 import { DateTime } from 'luxon';
 import { useTranslation } from 'react-i18next';
 import useGetAppSettings from 'src/hooks/useGetAppSettings';
-import illustration9 from '../../../images/drawkit/grape/drawkit-grape-pack-illustration-9.svg';
```

Comment:
I was going to suggest deleting this image from the repo too, but there are some illustrations that aren't being used, so it's probably fine.